File: Android4/PracticeImage10/app/src/main/java/com/example/practiceimage10/MainActivity.java
/* **********************************************
 * 프로그램명 :  activity-main.xml
 * 작성자 : 2020039003 김성연
 * 작성일 : 2022.04.03
 *프로그램 설명 :  버튼에도 이미지가 들어가도록 할 수 있다. 다음과 같이 버튼을 클릭하면 이미지가 10도씩 회전되도록 프로젝트를 작성하라.
 ************************************************/

package com.example.practiceimage10;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    Button btn1;
    ImageView img1;
    int trun = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setTitle("연습문제 4-9");

        btn1 = (Button) findViewById(R.id.btn1);
        img1 = (ImageView) findViewById(R.id.img1);

        btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                trun = trun+10;
                img1.setRotation(trun);
            }
        });
    }
}

File: Android3Test/app/src/main/java/com/example/androidtext/MainActivity.kt
/* **********************************************
 * 프로그램명 :  activity-main.xml
 * 작성자 : 2020039003 김성연
 * 작성일 : 2022.03.27
 *프로그램 설명 : Android Display Design
 ************************************************/

package com.example.androidtext

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent;
import android.net.Uri;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.RadioButton;
import android.widget.RadioGroup;
import android.widget.Toast;

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        getSupportActionBar()?.setDisplayHomeAsUpEnabled(true)
        getSupportActionBar()?.setIcon((R.drawable.ic_launcher))

        lateinit var button1: Button
        lateinit var editTest: EditText
        lateinit var button2 : Button

        button1 = findViewById(R.id.ch_display)
        editTest = findViewById(androidx.appcompat.R.id.edit_query)
        button2 = findViewById(R.id.homepagebtn)

        button1.setOnClickListener {
            Toast.makeText(
                applicationContext, editTest.text,
                Toast.LENGTH_SHORT
            ).show()
        }
        button2.setOnClickListener{
            var mIntent = Intent(Intent.ACTION_VIEW, Uri.parse(editTest.text.toString()))
            startActivity(mIntent)
        }

        lateinit var radioButton1: RadioButton
        lateinit var radioButton2: RadioButton
        lateinit var imageView: ImageView

        radioButton1 = findViewById(R.id.radio_1)
        radioButton2 = findViewById(R.id.radio_2)
        imageView = findViewById(R.id.image_view)

        radioButton1.setOnClickListener{
            imageView.setImageResource(R.drawable.ic_launcher)
        }

        radioButton2.setOnClickListener{
            imageView.setImageResource(R.drawable.ic_launcher2)
        }
    }
}

